main/opcodes/create-terminate-context.zkasm

```diff
@@ -537,12 +537,17 @@ opRETURNcreate:
 
     ; set bytecode length
     $ => A          :MLOAD(createContractAddress)
+
+    ; check bytecode first byte != 0xEF
+                    :CALL(checkBytecodeStartsEF)
```

Comment:
You can make the MLOAD and the CALL at the same line:
$ => A          :MLOAD(createContractAddress), CALL(checkBytecodeStartsEF)
One step less